### PR TITLE
visited links

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
https://jira.sng.sk/browse/NSNG-196

### CODE REVIEW

File: resources/js/stores/InteractionStore.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export const useInteractionStore = defineStore('InteractionStore', {
         interactions: useStorage('interactions', []),
         peekCodePanel: useStorage('peekCodePanel', true),
         cursor: useStorage('cursor', -1),
+        visited: useStorage('interaction.visited', []),
```
Comment (igor-kamil) [inline resources/js/stores/InteractionStore.js:10]: why to introduce new localstorage item, when there is already `interactions` (and also `stories`)
is it really necessary? 
can't we reuse what we have (and just to see, whether it contains interaction with given id)?

Comment (rastislav-chynoransky) [reply resources/js/stores/InteractionStore.js:10]: Sorry, you're not right. 🥲 
We're removing those objects on undo action so for the sake of simplicity I'd keep it like this.

**Comment by rastislav-chynoransky** [comment]
@igor-kamil Do you think `reset` should also clear visited stories or not?

**Comment by igor-kamil** [comment]
> @igor-kamil Do you think `reset` should also clear visited stories or not?

Sure

**Comment by rastislav-chynoransky** [comment]
Ok, I'd prefer to keep it how it was and not to mix the responsibilities of `storyStore`. It should work only as a cache and its behavior can get easily changed in the future.

**Review by igor-kamil — APPROVED** [review]
ok. now I see the reason why to use separated store just for `interaction.visited`. Time will tell. Keep it for now. Still we can change it to use only `interactions` - if we find it better suits our needs.

**Pull request merged**